release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -654,6 +661,18 @@ As an administrator, you can now install the Network Observability Operator to o
 
 For more information, see xref:../networking/network_observability/network-observability-overview.html#network-observability-overview[Network Observability].
 
+[id="ocp-4-12-nw-shiftstack-ipv6-secondary"]
+==== IPv6 for secondary network interfaces on {rh-openstack}
+
+IPv6 for secondary network intefaces is now supported in clusters that run on {rh-openstack}. 
+
+// TODO: For more information,see ${DOC_WHEN_MERGED} 
+
+[id="ocp-4-12-nw-shiftstack-udp-load-balancing"]
+==== UDP support for load balancers on {rh-openstack}
+
+Resulting from the switch to an external {rh-openstack} cloud provider, UDP is now supported for load balancers for clusters that run on that platform.
```

Comment:
```suggestion
Resulting from the switch to an external OpenStack cloud provider, UDP is now supported for load balancer type services for clusters that run on that platform.
```

Reply:
@eurijon I find this phrasing a little hard to parse. As an alternative, would you prefer "services that have the load balancer type" or "`LoadBalancer` services" here?

Reply:
+1 for "LoadBalancer services" :)

Reply:
```suggestion
Resulting from the switch to an external OpenStack cloud provider, UDP is now supported for `LoadBalancer` services for clusters that run on that platform.
```